static/.htaccess

```
@@ -19,11 +19,11 @@ RewriteRule ^docs/8.1/components/modeler/desktop-modeler/connect-to-camunda-clou
 RewriteRule ^docs/8.0/components/modeler/desktop-modeler/connect-to-camunda-cloud/(.*)$ /docs/8.0/components/modeler/desktop-modeler/connect-to-camunda-platform-8/$1 [R=301,L]
 
 # API Endpoints to API Orchestration
-RewriteRule ^docs/guides/orchestrate-api-endpoints(.*)$ /docs/guides/orchestrate-apis/$1 [R=301,L]
-RewriteRule ^docs/next/guides/orchestrate-api-endpoints(.*)$ /docs/next/guides/orchestrate-apis/$1 [R=301,L]
-RewriteRule ^docs/8.2/guides/orchestrate-api-endpoints(.*)$ /docs/8.2/guides/orchestrate-apis/$1 [R=301,L]
-RewriteRule ^docs/8.1/guides/orchestrate-api-endpoints(.*)$ /docs/8.1/guides/orchestrate-apis/$1 [R=301,L]
-RewriteRule ^docs/8.0/guides/orchestrate-api-endpoints(.*)$ /docs/8.0/guides/orchestrate-apis/$1 [R=301,L]
+RewriteRule ^docs/guides/orchestrate-api-endpoints(.*)$ /docs/guides/orchestrate-apis$1 [R=301,L]
+RewriteRule ^docs/next/guides/orchestrate-api-endpoints(.*)$ /docs/next/guides/orchestrate-apis$1 [R=301,L]
+RewriteRule ^docs/8.2/guides/orchestrate-api-endpoints(.*)$ /docs/8.2/guides/orchestrate-apis$1 [R=301,L]
+RewriteRule ^docs/8.1/guides/orchestrate-api-endpoints(.*)$ /docs/8.1/guides/orchestrate-apis$1 [R=301,L]
+RewriteRule ^docs/8.0/guides/orchestrate-api-endpoints(.*)$ /docs/8.0/guides/orchestrate-apis$1 [R=301,L]
```

@pepopowitz we need redirects on each supported version since this is already live? Is that thinking correct?

---

That's my understanding and interpretation, and I think it aligns with [this conversation about rewrite rules](https://github.com/camunda/camunda-platform-docs/issues/1397#issuecomment-1677542780). Once those versions are archived/unsupported, I'll be removing those versioned rules anyway.

Reviewing that linked comment though, I do think we shouldn't worry about `/next` redirects, and should remove that line. I overlooked that in #2541, but that would be cleaned up in the work we do for #1397.

---

```suggestion
RewriteRule ^docs/8.2/guides/orchestrate-api-endpoints(.*)$ /docs/8.2/guides/orchestrate-apis$1 [R=301,L]
RewriteRule ^docs/8.1/guides/orchestrate-api-endpoints(.*)$ /docs/8.1/guides/orchestrate-apis$1 [R=301,L]
RewriteRule ^docs/8.0/guides/orchestrate-api-endpoints(.*)$ /docs/8.0/guides/orchestrate-apis$1 [R=301,L]
```

See my previous comment -- since `/next` is considered work-in-progress, I don't think we need to worry about redirects there.